### PR TITLE
Initial layout

### PR DESCRIPTION
**Pull request by lxcid**
This provide an initial layout with `styled-components` and `react-router-dom` setup.

I use kinda similar folder structure described here but not 100% yet, I can iterate to 100% if we are agreeable to use the same folder structure.

https://gist.github.com/ryanflorence/daafb1e3cb8ad740b346

It have 2 screen, one home and one listing.

I setup a very basic skeletal that follow the figma design.

The main layout is done with grid, subsequently we just use flexbox. I'm not strong with CSS so it looks kinda sucky at the moment.

Screen Shot 2020-04-01 at 10 32 45 PM

### CODE REVIEW

**Comment by lxcid** [comment]
To try it out:

- go to `ui` directory
- type `yarn install`
- type `yarn start`

**Comment by lxcid** [comment]
Current status:

Screen Shot 2020-04-02 at 9 50 19 AM

File: ui/src/screens/Posts/components/Post.tsx [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+  border-radius: 14px;
+  overflow: hidden;
+`;
+
+type SelectionIndicatorProps = {
+  active: boolean,
+};
+const SelectionIndicator = styled.div`
+  width: 12px;
+  background-color: ${(props: SelectionIndicatorProps) => props.active ? '#3498DB' : 'none'}
```
Comment (SheldonCheng) [inline ui/src/screens/Posts/components/Post.tsx]: `none` should be `transparent`

**Review by SheldonCheng — APPROVED** [review]

**Comment by SheldonCheng** [comment]
I think the layout needs some work on responsive but we can improve it later. Otherwise just a little typo!

**Comment by lxcid** [comment]
Thanks @SheldonCheng! I'll make the change as suggested and merge!

File: ui/src/screens/Posts/components/Post.tsx (hunk 1 above)
Comment (lxcid) [reply ui/src/screens/Posts/components/Post.tsx]: fixed in d5bbf9f

**Pull request merged**